out.py

```diff
@@ -15,14 +15,15 @@ class _CursorInfo(ctypes.Structure):
 
 
 def error(message: str, context: Any = __UNIT) -> None:
-    print(message, file=sys.stderr)
+    print(ERROR + message + RESET, file=sys.stderr)
```

Comment:
print the error output using the helper out.println function
adding an optional `file` parameter might be required.